### PR TITLE
mdd-272167-Added-Single-Source-StepArea/Line&&SplineArea/Spline

### PR DESCRIPTION
**Pull request by mddifilippo89**
mdd-272167-Added-Single-Source-

StepArea/Line &&
SplineArea/Spline

### CODE REVIEW

File: samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        _data.Add(new EnergyChartData() { Year = "2015", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2016", Europe = 13 });
+        _data.Add(new EnergyChartData() { Year = "2017", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2018", Europe = 40 });
+        _data.Add(new EnergyChartData() { Year = "2019", Europe = 80 });
+
+        this.DataSource = _data;
+    }
+
+    public class EnergyChartData
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor:69]: remove this class and add [Services\EnergyRenewableData.cs](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/category-chart/line-chart-single-source/Services/EnergyRenewableData.cs) file

File: samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+        _data.Add(new EnergyChartData() { Year = "2015", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2016", Europe = 13 });
+        _data.Add(new EnergyChartData() { Year = "2017", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2018", Europe = 40 });
+        _data.Add(new EnergyChartData() { Year = "2019", Europe = 80 });
+
+        this.DataSource = _data;
+    }
+
+    public class EnergyChartData
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor:69]: remove this class and add [Services\EnergyRenewableData.cs](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/category-chart/line-chart-single-source/Services/EnergyRenewableData.cs) file

File: samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+        _data.Add(new EnergyChartData() { Year = "2015", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2016", Europe = 13 });
+        _data.Add(new EnergyChartData() { Year = "2017", Europe = 78 });
+        _data.Add(new EnergyChartData() { Year = "2018", Europe = 40 });
+        _data.Add(new EnergyChartData() { Year = "2019", Europe = 80 });
+
+        this.DataSource = _data;
+    }
+
+    public class EnergyChartData
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor:69]: remove this class and add [Services\EnergyRenewableData.cs](https://github.com/IgniteUI/igniteui-blazor-examples/blob/vnext/samples/charts/category-chart/line-chart-single-source/Services/EnergyRenewableData.cs) file

File: samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+    protected override void OnInitialized()
+    {
+        CategoryChartModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+        LegendModule.Register(IgniteUIBlazor);
+        InitData();
+    }
+
+    private void InitData()
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor:51]: remove this function

File: samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+    protected override void OnInitialized()
+    {
+        CategoryChartModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+        LegendModule.Register(IgniteUIBlazor);
+        InitData();
+    }
+
+    private void InitData()
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor:51]: remove this function

File: samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+
+    protected override void OnInitialized()
+    {
+        CategoryChartModule.Register(IgniteUIBlazor);
+        DataChartInteractivityModule.Register(IgniteUIBlazor);
+        LegendModule.Register(IgniteUIBlazor);
+        InitData();
+    }
+
+    private void InitData()
```
Comment (HUSSAR-mtrela) [inline samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor:51]: remove this function

**Review by HUSSAR-mtrela — CHANGES_REQUESTED** [review]

File: samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor (hunk 1 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor]: see commit 3

File: samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor (hunk 2 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor]: see commit 3

File: samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor (hunk 3 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor]: commit 3

File: samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor (hunk 4 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/spline-area-single-source/Pages/SplineAreaSingleSource.razor]: commit 3

File: samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor (hunk 6 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/step-line-single-source/Pages/StepLineSingleSource.razor]: commit 3

File: samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor (hunk 5 above)
Comment (mddifilippo89) [reply samples/charts/category-chart/spline-single-source/Pages/SplineSingleSource.razor]: commit 3

**Review by HUSSAR-mtrela — APPROVED** [review]

**Pull request merged**